lib/feature/pokemon_details_page.dart/pokemon_details_page.dart

```diff
@@ -1,24 +1,176 @@
+                              child: Row(
+                                children: [
+                                  Container(
+                                    decoration: BoxDecoration(
+                                      color: Colors.grey, // change color later
+                                      borderRadius: BorderRadius.circular(50),
+                                    ),
+                                    child: Padding(
+                                      padding: EdgeInsets.fromLTRB(10, 3, 10, 3),
+                                      child: Text(
+                                        '${pokemon?.types?.first.specificType?.name ?? ''}',
+                                        style: TextStyle(
+                                          fontSize: 15,
+                                          color: Colors.white,
+                                        ),
+                                      ),
+                                    ),
+                                  ),
+                                  Visibility(
+                                    visible: pokemon?.types?.last.specificType?.name ==
+                                            pokemon?.types?.first.specificType?.name
+                                        ? false
+                                        : true,
+                                    child: Container(
+                                      decoration: BoxDecoration(
+                                        color: Colors.grey, // change color later
+                                        borderRadius: BorderRadius.circular(50),
+                                      ),
+                                      child: Padding(
+                                        padding: EdgeInsets.fromLTRB(10, 3, 10, 3),
+                                        child: Text(
+                                          '${pokemon?.types?.last.specificType?.name == pokemon?.types?.first.specificType?.name ? '' : pokemon?.types?.last.specificType?.name}',
+                                          style: TextStyle(
+                                            fontSize: 15,
+                                            color: Colors.white,
+                                          ),
+                                        ),
+                                      ),
+                                    ),
+                                  )
+                                ],
+                              ),
+                            ),
+                          ],
+                        ),
+                      ),
+                      Container(
+                        height: 200,
+                        child: Image.network(
+                          '$pokemonImgUrl${pokemon?.id}.png',
+                          fit: BoxFit.cover,
+                          errorBuilder: (context, error, stackTrace) {
```

Comment:
for callbacks that contains parameters that we don't use, assign it to `_`. so in this case `(_, __, ___)`